packages/wallet/src/redux/protocols/direct-funding/state.ts

```diff
@@ -105,7 +108,10 @@ export type DirectFundingState =
   | WaitForFundingConfirmation
   | ChannelFunded;
 
-export function initialDirectFundingState(action: DirectFundingRequested) {
+export function initialDirectFundingState(
```

Comment:
I wonder if we should make a convention around naming. In the `TransactionSubmission` protocol, there is an `initialize` function, which seems reasonable.